client-side/src/components/profileComponents/addProfileComponent.jsx

```diff
@@ -94,7 +94,7 @@ import '../../styles/profilePageStyle.scss';
       }
       dispatch(addProfile(ProfileNew));
       setTimeout(() => navigate(0), 2000);
-      handleClose();
+      // handleClose();
```

Comment:
this is not my component
It gives an error when running the project
This function does not exist in the component